statistics/histogram.go

```diff
@@ -808,12 +830,31 @@ type Column struct {
 	ErrorRate
 	Flag           int64
 	LastAnalyzePos types.Datum
+	StatsVer       int64 // StatsVer is the version of the current stats, used to maintain compatibility
```

Comment:
`StatsVer` seems confused, so how about renaming it to `ConsiderTopN` or `UseTopN`?